openslides/global_settings.py

```diff
@@ -71,7 +67,7 @@
     'django.middleware.locale.LocaleMiddleware',
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
-    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'openslides.users.anonymous.AuthenticationMiddleware',
```

Comment:
Hm. I think we need a generic term for what is done in this file resp. in the middleware, that means for both: anonymous "authentication" and authetication with credentials. such a generic term can be "auth" or "authentication" or may be something else but not "anonymous".